templates/3a_monitor_MIBI_run.ipynb

```diff
@@ -1,12 +1,5 @@
 {
```

Comment:
<p>I don't see any circumstance where a non-expert user will be modifying the paths that data gets saved to. Likewise telling the user about the dependencies between FOV and run callbacks. This explanation should be laser-focused on just the information users need to understand what needs to be modified, which is just the list in run_callbacks and fov_callbacks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>